### DISCUSSION TITLE
No digits in value  field in Overview.

### DISCUSSION THREAD

**Discussion by AndyAnka** [jomjol/AI-on-the-edge-device#2538] [General] [answered]
The ESP32-CAM presents correctly values in previous value, raw value, value status, process state and system system info but data in value field. Has anyone any idea whats wrong.

**Comment by AndyAnka** [comment]
![IMG_0052](https://github.com/jomjol/AI-on-the-edge-device/assets/42616240/9abf6440-45ca-48aa-8115-0c07f61db93a)

**Comment by friedpa** [comment]
You can see in the "Value Status" that you have a "Negative Rate" clssification of the Raw Value (New Value smaller than the old Value), therfore a missreading -> No Value to send/ display -> empty "Value" field

**Comment by Slider0007** [comment, marked as answer]
Using Settings -> Set previous value, you can set the previous value to the actual value to reset the rate checking algo.

**Comment by AndyAnka** [comment]
Thank’s for your answers! Now it runs perfectly!
